### DISCUSSION TITLE
Need help for hide cover image from single.html

### DISCUSSION THREAD

**Discussion by khoile2002** [gethinode/hinode#1286] [Help] [answered]
Thanks for a wonderful theme. I have an issue that I need help with. I want to hide the cover image from the detail single page ( single.html). Where should I edit with code. Many Thank!

**Comment by khoile2002** [comment, marked as answer]
Thank you, I have resolved the issue. ( with claude ai )
Add a condition at the start of thumbnail.html to skip rendering based on a parameter, e.g., noCover:
`{{- $page := .page -}}
{{ if not $page.Params.noCover }}
    {{- $wrapper := printf "img-wrap" -}}
    ...
    {{ if $thumbnail -}}
        {{- partial "assets/image.html" (dict "url" $thumbnail "ratio" $ratio "wrapper" $wrapper "class" $class "title" $page.Params.title "caption" $credits "figclass" $figclass "priority" "high") -}}
    {{ end -}}
{{ end }}
`
Add noCover: true to the front matter of any page where you don't want the cover image to appear.

**Reply by markdumay** [reply]
I'm impressed with the solution presented by Claude AI! Glad to hear you like the theme!
